ocs_ci/helpers/dr_helpers.py

```diff
@@ -73,3 +76,177 @@ def relocate(preferred_cluster, drpc_name, namespace):
     drpc_obj.wait_for_phase(constants.STATUS_RELOCATED)
 
     config.switch_ctx(prev_index)
+
+
+def check_mirroring_status_ok(replaying_images=None):
+    """
+    Check if mirroring status has health OK and expected number of replaying images
+
+    Args:
+        replaying_images (int): Expected number of images in replaying state
+
+    Returns:
+        bool: True if status contains expected health and states values, False otherwise
+
+    """
+    cbp_obj = ocp.OCP(
+        kind=constants.CEPHBLOCKPOOL,
+        resource_name=constants.DEFAULT_CEPHBLOCKPOOL,
+        namespace=constants.OPENSHIFT_STORAGE_NAMESPACE,
+    )
+    mirroring_status = cbp_obj.get().get("status").get("mirroringStatus").get("summary")
+    logger.info(f"Mirroring status: {mirroring_status}")
+    keys_to_check = ["health", "daemon_health", "image_health", "states"]
+    for key in keys_to_check:
+        if key != "states":
+            expected_value = "OK"
+            value = mirroring_status.get(key)
+        elif key == "states" and replaying_images:
+            # Replaying images count can be higher due to presence of dummy images
+            expected_value = range(replaying_images, replaying_images + 3)
+            value = mirroring_status.get(key).get("replaying")
```

Comment:
we can keep the key as "states" here as that's the only condition when this value assignment will happen.

Reply:
Done.